sass/base/_typography.scss

```diff
@@ -131,9 +131,10 @@ h1 {
 }
 
 .page-title {
+  margin-top: 0;
```

Comment:
This was not done in v1 so the page title has a top margin, and will result in different vertical placement that will fail all VRTs unless subtheme has overriding rules.